src/Tizen.Maps/Tizen.Maps/MapView.cs

```diff
@@ -59,6 +59,7 @@ public class MapView : Layout, IDisposable
         /// <exception cref="System.InvalidOperationException">Thrown when a native operation failed to allocate memory, and connect to the service.</exception>
         public MapView(EvasObject parent, MapService service) : base(parent)
         {
+            Log.Info(string.Format("ViewReady event callback is unset"));
             handle = new Interop.ViewHandle(service.handle, this);
             service.handle.HasOwnership = false;
             Log.Info(string.Format("MapView is created"));
```

Comment:
The behavior of 'string.Format(string, params object[])' could vary based on the current user's locale settings. Replace this call in 'MapView.MapView(EvasObject, MapService)' with a call to 'string.Format(IFormatProvider, string, params object[])'.